wiki/Client/Interface/en.md

```diff
@@ -284,31 +287,34 @@ Below are the results screens of the other game modes.
+- \[2\] `Save replay to Replays folder`: You can watch the replay later either by opening it from a local leaderboard, or by going to `Replays` directory and double clicking it.
+- \[3\] `Add as online favourite`: Include the beatmap into your list of favourites, which is located on your osu! profile page under the `Beatmaps` section.
+- \[4\] Local leaderboard: All your results are stored on your computer. To see them, navigate to the [song selection screen](#song-selection-screen) and select `Local Rankings` from the drop-down menu above the scoreboard.
+- \[5\] `Beatmap Ranking` section. Available only for maps with online leaderboards ([qualified](/wiki/Beatmap/Category#qualified), [ranked](/wiki/Beatmap/Category#ranked), or [loved](/wiki/Beatmap/Category#loved)). You also need to be online to see this section. See below for more details.
+- \[6\] `Overall Ranking` section. It's available only for beatmaps with online leaderboards. You also need to be online to see this section. See below for more details.
+- \[7\] Information about the beatmap with its play count and pass rate.
+- \[8\] Beatmap rating. Use your personal discretion based on whether you enjoyed the beatmap or not. Best left alone if you can't decide.
+- \[9\] Click here to return to the song selection screen.
+
+---
+
+The categories in the ranking panel are as follows:
+
+| Category | Beatmap Ranking | Overall Ranking |
+| :-: | :-- | :-- |
+| `Overall` | Your position on the map's leaderboard, where you compete against players that used [Mods](/wiki/Game_modifier), even if you didn't use any yourself. | Your global ranking in the world. |
+| `Accuracy` | How [precisely](/wiki/Gameplay/Accuracy) did you play the beatmap. Will only be counted when your old score is surpassed. | Your average [accuracy](/wiki/Gameplay/Accuracy#accuracy) over all beatmaps you have played. |
+| `Max Combo` | Your longest combo on the map you played. | The longest combo over all beatmaps you have played. |
+| [`Ranked Score`](/wiki/Gameplay/Score/Ranked_score) | Your [best result](/wiki/Gameplay/Score/Ranked_score) on the beatmap. | The number of points earned from all ranked beatmaps that you have ever played, with every map being counted exactly once. |
+| [`Total Score`](/wiki/Gameplay/Score/Total_score) | Not taken into account, since it does not affect your position in online rankings. | Same as ranked score, but it takes into account all beatmaps available on the osu! website, and also underplayed or failed beatmaps. This counts towards your [level](/wiki/Gameplay/Score/Total_score#level). |
+| `Performance` | The amount of [unweighted pp](/wiki/Performance_points#why-didnt-i-gain-the-full-amount-of-pp-from-a-map-i-played?) you would receive for the play. | Displays your total amount of performance points, and also how much pp the submitted play was worth. |
+
+### Medals
 
```

Comment:
replace the paragraph on line 321 below with `Sometimes, when specific conditions are met, you may receive a medal.`

also the medal image is a spoiler for a hush-hush medal, so it should probably be replaced (can do that later)

```suggestion

*Main page: [Medals](/wiki/Medals)*

```

Reply:
I have a screenshot of the 5000 plays medal, but it's unfortunately with a custom skin 🤔